PORT-2479 Plenty of docs fixes and updates (#151)
port-labs/port-docs: docs/software-catalog/blueprint/tutorial.md

To update a Blueprint, you can do either of the following:

> - Click the 3 dots (`...`) button and then click the `Properties` button in the Blueprints graph;

Reviewer: ```suggestion
- In the Blueprints graph, click on the 3 dots (`...`) button and after it the `Properties` button;
```

- Make an HTTP PUT request to the URL: `https://api.getport.io/v1/{blueprint_identifier}`
  - A PUT request has the same body as a POST request, it will simply overwrite the Blueprint with the new data provided.

![Blueprints Graph edit button marked](../../../static/img/software-catalog/blueprint/blueprintGraphEditButtonMarked.png)

### Update Blueprint identifier

> In order to change the identifier of a Blueprint you can use one of the following options:

Reviewer: ```suggestion
There are two methods to to change a Blueprint's identifier:
```

#### From the UI

> 1. In the Blueprints graph, on the Blueprint you want to update, click the 3 dots (`...`) button and then click the `Properties` button;
> 2. Update the value of the `identifier` key to the new identifier;
> 3. Click `Save`.

Reviewer: ```suggestion
1. In the desired Blueprints graph, click on the 3 dots (`...`) button and then on the `Properties` button;
2. Update the value of the `identifier` key to the new identifier;
3. Click `Save`.
```

#### PUT request

> Make an HTTP PUT request to the URL `https://api.getport.io/v1/blueprints/{old_blueprint_identifier}` with the Blueprint definition JSON, after updating the value of the `identifier` key in the request body to the new identifier.

Reviewer: I don't understand if first there needs to be a HTTP request or a change in the identifier key. WHatever is first, should appear first in the sentence.
